docs/sample-config.yaml

```diff
@@ -0,0 +1,72 @@
+rules:
+# Each rule represents a some naming and discovery logic.
+# Each rule is executed independently of the others, so
+# take care to avoid overlap.  As an optimization, rules
+# with the same `seriesQuery` but different
+# `naming` or `seriesFilters` will use only one query to
+# Prometheus for discovery.
+
+# some of these rules are taken from the "default" configuration, which
+# can be found in pkg/config/default.go
+
+# this rule matches cumulative cAdvisor metrics measured in seconds
+- seriesQuery: '{__name__=~"^container_.*",container_name!="POD",namespace!="",pod_name!=""}'
+  resources:
+    # skip specifying generic resource<->label mappings, and just
+    # attach only pod and namespace resources by mapping label names to group-resources
+    overrides:
+      namespace: {resource: "namespace"},
+      pod_name: {resource: "pod"},
```

Comment:
honestly I think this is a change we should just make upstream, it's annoying that cadvisor labels are not following the official guidelines for instrumentation, it makes joining metrics unnecessarily hard

Reply:
agreed, I've proposed it before.  It'll take some doing though :-/

Reply:
I've heard "metrics are completely left out of stability guarantees for Kubernetes" often enough that I think it's a legit change for the better. Let's bring it up in the next sig meeting.